db/event_manager_test.go

```diff
@@ -39,26 +38,26 @@ func (th *TestingHandler) HandleEvent(event Event) {
 
 		doc := dsceEvent.Doc
 
-		goassert.Equals(th.t, len(doc), 5)
+		assert.Equal(th.t, 5, len(doc))
 
 		state := doc["state"]
 
 		//state must be online or offline
-		goassert.True(th.t, state != nil && (state == "online" || state == "offline"))
+		assert.True(th.t, state != nil && (state == "online" || state == "offline"))
 
 		//admin interface must resolve to a a valis tcp address
 		adminInterface := (doc["admininterface"]).(string)
 
 		_, err := net.ResolveTCPAddr("tcp", adminInterface)
 
-		goassert.Equals(th.t, err, nil)
+		assert.Equal(th.t, nil, err)
```

Comment:
assert.NoError is preferable here and below.

Reply:
Thank you Adam. This has been corrected.